indico/modules/events/abstracts/templates/_abstract.html

```diff
@@ -270,10 +270,19 @@ <h3 class="abstract-title">
                 <div class="i-timeline-item">
                     <div class="i-timeline-item-label action"></div>
                     <div class="i-timeline-item-box header-indicator">
-                        <div class="i-box-header">
-                            {% trans name=abstract.judge.name, action=abstract.state.title|lower, date=abstract.judgment_dt|format_human_date -%}
-                                <strong>{{ name }}</strong> {{ action }} this abstract on {{ date }}.
-                            {%- endtrans %}
+                        <div class="i-box-header flex">
+                            <div class="stretch">
+                                {% trans name=abstract.judge.name, action=abstract.state.title|lower, date=abstract.judgment_dt|format_human_date -%}
+                                    <strong>{{ name }}</strong> {{ action }} this abstract on {{ date }}.
+                                {%- endtrans %}
+                            </div>
+                            {% if abstract.event_new.can_manage(session.user) %}
+                                <div>
+                                    <a href="{{ url_for('.reset_abstract_jugment', abstract) }}"
+                                       class="i-button-icon icon-remove" title="{% trans %}Reset judgement{% endtrans %}"
+                                       data-confirm="{% trans %}Do you really want to reset the judgment? This operation is irreversible.{% endtrans %}"></a>
```

Comment:
OK, I agree. But I mostly mentioned this because of the `class` and `title` being in the same line. No strong opinion, just a personal preference.